juju/juju
# [JUJU-1280] new yaml color scheme juju config

*Add vim default yaml color scheme for  `juju config yaml` output*

## Checklist


## QA steps

```sh
  juju bootstrap localhost overlord
  juju add-model lxd-model 
  juju deploy mediawiki
  juju deploy mysql --series=trusty
  juju deploy haproxy
  juju add-relation mediawiki:db mysql
  juju add-relation mediawiki haproxy
  juju expose haproxy
  juju add-unit -n 2 mediawiki
  juju config haproxy  --color
```
*The yaml output should be colored and should be valid yaml*
